src/extra/treemacs-all-the-icons.el

```diff
@@ -0,0 +1,240 @@
+;;; treemacs-all-the-icons.el --- all-the-icons integration for treemacs -*- lexical-binding: t -*-
+
+;; Copyright (C) 2020 Alexander Miller
+
+;; Author: Alexander Miller <alexanderm@web.de>
```

Comment:
Don't know what the proper etiquette about these things is like, but you wrote (the first version of) this theme, so IMO  you should put yourself down as the author as well. At least that's how it went in treemacs-perspective.el.

Reply:
I had the same doubt :sweat_smile:, I think it makes sense.